llvm/lib/IR/Verifier.cpp

```diff
@@ -808,6 +808,10 @@ void Verifier::visitGlobalValue(const GlobalValue &GV) {
           "visibility must be dso_local!",
           &GV);
 
+  if (GV.isTagged()) {
```

Comment:
Is this really a verifier issue? If the frontend wants to tag a global in a section (e.g. because it is known to be safe), shouldn't it be able to?

Reply:
Ah okay, if it's a limitation in the existing codegen code that's fine and we can relax it later if needed.

@arichardson yes, we're aware of the reason for this restriction, the question was whether it should be enforced here as well as in the frontend.